tendrl-dashboard.spec

```diff
@@ -8,16 +8,57 @@ Group:   Applications/System
 Source0: %{name}-%{version}.tar.gz
 URL: https://github.com/Tendrl/dashboard
 
-Requires: tendrl-api
+BuildRequires: npm
+BuildRequires: nodejs-packaging
+BuildRequires: fontconfig
 
 %description
 Contains the JavaScript GUI content for the Tendrl front-end components
 (dashboard, login screens, administration screens)
 
 %prep
-%autosetup
+%setup -q -n %{name}-%{version}
+%nodejs_fixdep foomodule
 
 %build
+npm install gulp
+npm install gulp-concat
+npm install gulp-rename
+npm install gulp-cssimport
+npm install gulp-sass
+npm install gulp-minify-css
+npm install gulp-postcss
+npm install gulp-eslint
+npm install gulp-debug
+npm install gulp-if
+npm install gulp-ignore
+npm install gulp-inject
+npm install gulp-load-plugins
+npm install gulp-match
+npm install gulp-ng-annotate
+npm install gulp-sourcemaps
+npm install gulp-uglify
+npm install autoprefixer-core
+npm install karma
+npm install run-sequence
+npm install merge-stream
+npm install numeral
+npm install eslint
+npm install resource
+npm install copy
+npm install sass
+npm install preload
+npm install jsbundle
+
+./node_modules/.bin/gulp release
+
+%{__rm} -f .gitignore
+#
+# Fedora guidlines do not allow bundled modules
```

Comment:
"guidlines" -> "guidelines"

And now Fedora does allow bundled libraries, but it would be a good idea to add `Provides: bundled(foo)` for each bundled library so this can be tracked by the security team.